tests/datatypes/test_frozenset.py

```diff
@@ -2,7 +2,33 @@
 
 
 class FrozensetTests(TranspileTestCase):
-    pass
+    def test_creation(self):
+        # Empty dict
+        self.assertCodeExecution("""
+            x = frozenset()
+            print(x)
+            """)
+
+        # Set constant
+        self.assertCodeExecution("""
+            x = {'a'}
+            print(x)
+            """)
+
+        self.assertCodeExecution("""
+            x = frozenset(['a'])
+            print(x)
+            """)
+
+        self.assertCodeExecution("""
+            x = frozenset((1, 2))
```

Comment:
This has tested list and tuple, should there be a test for a set (and frozenset) as input too? We should also test whether changing the source set/array/tuple affects the newly created frozenset.